onboard/nav/stateMachine.cpp

```diff
@@ -405,6 +493,25 @@ bool StateMachine::isWaypointReachable( double distance )
     return isLocationReachable( mPhoebe, mRoverConfig, distance, mRoverConfig["navThresholds"]["waypointDistance"].GetDouble());
 } // isWaypointReachable
 
+bool StateMachine::isRadioRepeaterDrop() const
+{
+    if ( mPhoebe->roverStatus().currentState() != NavState::RadioRepeaterTurn &&
```

Comment:
This is really complicated for an if statement so please comment what you're checking